pkg/tools/cli/loadtest/usecase/load.go

```diff
@@ -70,11 +72,16 @@ func (r *run) PreStart(ctx context.Context) (err error) {
 
 // Start runs load test and returns error if occurred.
 func (r *run) Start(ctx context.Context) (<-chan error, error) {
-	rech, err := r.client.StartConnectionMonitor(ctx)
-	if err != nil {
-		return nil, err
-	}
+	// TODO: related to #557
+	/*
+		rech, err := r.client.StartConnectionMonitor(ctx)
+		if err != nil {
+			return nil, err
+		}
+	*/
+
```

Comment:
sorry. could you please use errors.Is to compare error?

https://github.com/vdaas/vald/blob/b541c1adbe6ae3db8d7394a652ae603cf148f242/pkg/tools/cli/loadtest/usecase/load.go#L97


```
err != nil && errors.Is(err, context.Canceled)
```

Reply:
This is based on https://github.com/vdaas/vald/blob/76908b610be98961163ee1b826e797370c5e999e/internal/client/discoverer/discover.go#L119-L136 and some similar lines, so I think it's better to match it.
How about it?

Reply:
I think original implementation is not correct.
in go, if we want to compare error, we use `errors.Is`

Reply:
I see. I fix it now.
By the way, is `!errors.Is(err, context.Canceled)` correct?

Reply:
yes, your example is correct (!errors.Is(err, context.Canceled))!